nilearn/_utils/cache_mixin.py

```diff
@@ -117,6 +119,10 @@ def cache(func, memory, func_memory_level=None, memory_level=None,
         be cached or not (if user_memory_level is equal of greater than
         func_memory_level the function is cached)
 
+    shelve: boolean,
+        Whether to return a MemorizedResult, callable by a .get() method,
```

Comment:
Say that it's a joblib MemorizedResult, so that people who don't know what it is can Google.